iam-login-service/src/test/java/it/infn/mw/iam/test/registration/RegistrationUnprivilegedTests.java

```diff
@@ -193,7 +205,7 @@ public void testUsernameAlreadyTaken() throws Exception {
   }
 
   @Test
-  public void testCreateRequestWithoutNotes() throws Exception {
+  public void testCreateRequestWithMandatoryNotesField() throws Exception {
```

Comment:
We need also to test when notes is not mandatory and we can do this on other classes that has @SpringBootTest(properties = { "iam.registration.fields.notes.field-behaviour=optional" }) on top or @SpringBootTest(properties = { "iam.registration.fields.notes.field-behaviour=hidden" }). This last value can be tested in the same way optional is. You can also test them by testing only the Validators or the Service, without all the mvc part.

Reply:
but if you prefer the mvc tests are ok :)

Reply:
but you have already done something similar here https://github.com/indigo-iam/iam/pull/782/files#diff-015aa3049cced12a11b63113340fe4f005b8a504559522e34b3818d667e6043aR106 (but optional tests are missing, or am i missing them?)

Reply:
Removed that Enrico 👍 . And implemented that in "you can also test them by testing only the Validators or the Service" part. 9d06dd3a089c4172d02077a4540b1c783a355130